[ADD] point_of_sale: add Mollie terminal doc (#17503)
odoo/documentation: content/applications/sales/point_of_sale/payment_methods/terminals/mollie.rst

======
Mollie
======

**Mollie** is a payment service that offers payment solutions through physical :doc:`payment terminals <../terminals>`,
tap terminals, and terminal apps for Android and iOS devices.

.. note::
   - Mollie payment terminals do not require an IoT Box to operate.

.. seealso::

>    - `List of supported countries <https://help.mollie.com/hc/en-us/articles/33911501243154-In-person-payments-supported-countries>`_

Reviewer: Could you please remove the bullet points within the **Note** and the **See also** banners? Since each only has one list item, like in the next section's **Warning**.

Mollie configuration
====================

To configure a Mollie terminal, create a `Mollie account <https://my.mollie.com>`_, then follow these steps:

> #. Go to the :guilabel:`In-person payments` section in the left sidebar to access the list of terminals.

Reviewer 2: ```suggestion
#. In the left menu, go to :menuselection:`In-person payments`.
```

> #. Select the desired payment terminal from the list.

Reviewer 2: When I tested this from scratch, I do not have a list of terminals; instead, I have to click *Buy a terminal* and then I can click one of the available terminals to buy one.

I assume that, if a terminal has already been purchased, I assume only that terminal shows up in the list under the In-person payments menu? and the list would contain as many entries as the number of terminals bought under the account? And hwo often do people actually have several terminals in that list?

Author: Yes, once any terminal is linked to the account, there will be a list like so:
<img width="1057" height="382" alt="image" src="https://github.com/user-attachments/assets/bbe92040-ef04-4a08-8180-5869249845b1" />

I've invited you to the Odoo org so you can see the same terminals as us.

> #. Click the :guilabel:`Terminal information` tab in the top menu.

Reviewer 2: ```suggestion
#. Click :guilabel:`Terminal information` in the top menu.
```

> #. Copy the :guilabel:`Terminal ID` and keep it handy for the :ref:`Odoo POS configuration <pos/mollie/pos_configuration>` process.

Reviewer 2: ```suggestion
#. Copy the :guilabel:`Terminal ID`; it is required for the :ref:`Odoo POS configuration <pos/mollie/pos_configuration>` process.
```
*keep it handy* is too informal for documentation

> #. Click the :guilabel:`Developers` icon in the bottom left corner.

Reviewer 2: ```suggestion
#. Click the :guilabel:`Developers` icon at the bottom of the left menu.
```

> #. Copy the :guilabel:`Live API key` and keep it handy for the :ref:`Odoo POS configuration <pos/mollie/pos_configuration>` process.

Reviewer 2: ```suggestion
#. Copy the :guilabel:`Live API key`; it is required for the :ref:`Odoo POS configuration <pos/mollie/pos_configuration>` process.
```
same as above

.. warning::

>    Treat the :guilabel:`Live API key` as a password and keep it secure, as it provides access to your Mollie account.

Reviewer 2: ```suggestion
   The :guilabel:`Live API key` provides access to the Mollie account and must be kept secure.
```

.. note::

>    To set up a Mollie terminal, find all the available options on the `Mollie website <https://www.mollie.com/products/pos-payments>`_.

Reviewer 2: ```suggestion
   Refer to the `Mollie website <https://www.mollie.com/products/pos-payments>`_ for a full list of supported terminals and setup instructions.
```
this describes the link a bit better I think

.. _pos/mollie/pos_configuration:

Odoo POS configuration
======================

To connect the Mollie terminal with Odoo Point of Sale, follow these steps:

#. Go to :menuselection:`Point of Sale --> Configuration --> Payment Methods` and :doc:`create a
   payment method <../../payment_methods>`.
#. Set the :guilabel:`Journal` field to :guilabel:`Bank`.
#. Select the desired point of sale in the :guilabel:`Point of Sale` field.
#. Set the :guilabel:`Integration` field to :guilabel:`Terminal`.
#. Click :guilabel:`Activate` or :guilabel:`Setup` under the Mollie integration from the list of providers if needed.
#. Optionally, set the :guilabel:`Integrate with` field to :guilabel:`Mollie`.
#. Set the :guilabel:`Mollie Payment Provider` field to :guilabel:`Mollie`.

> #. Paste the Terminal ID into the :guilabel:`Mollie Terminal ID` field.

Reviewer 2: I'd add a ref to the Mollie configuration section here for *terminal ID* (and remove the capital T):

```suggestion
#. Paste the :ref:`terminal ID <pos/mollie/mollie_configuration>` into the :guilabel:`Mollie Terminal ID` field.
```

> #. Save the payment method.

Reviewer 2: I don't think this is needed; I'd remove this step

> #. Click the :icon:`fa-arrow-right` (:guilabel:`Internal link`) icon next to the :guilabel:`Mollie Payment Provider` field.

Reviewer 2: ```suggestion
#. Hover over the :guilabel:`Mollie Payment Provider` field and click the :icon:`fa-arrow-right` (:guilabel:`Internal link`) icon.
```

> #. Paste the API key into the :guilabel:`API Key` field. Leave the :guilabel:`State` field

Reviewer 2: same here with the ref to the previous section

>    on :guilabel:`Disabled` if the payment provider is not used for online payments.

Reviewer 2: I'd add a doc link to the [Mollie payment provider doc](https://www.odoo.com/documentation/master/applications/finance/payment_providers/mollie.html):

```suggestion
   on :guilabel:`Disabled` if the Mollie payment provider is not used for :doc:`online payments <add the link here>`.
```

======
Mollie
======

Connecting a **Mollie** :doc:`payment terminal <../terminals>` allows you to offer a fluid payment

> flow to your customers and ease the work of your cashiers.

Reviewer: ```suggestion
**Mollie** is a payment service that offers payment solutions through physical :doc:`payment terminals <../terminals>`, tap terminals, and terminal apps for Android and iOS devices.
```

.. note::

>    - Mollie payment terminals do not require an :doc:`IoT Box </applications/general/iot>` to

Reviewer: Since it does not require an IoT box, I think we can remove the doc link here. 
```suggestion
   - Mollie payment terminals do not require an IoT Box to
```

     operate.

>    - The Mollie Tap app allows a smartphone with an NFC chip to be used as a payment terminal.

Reviewer: Tables cannot be used as payment terminals, right? I'm asking because Mollie's website never specifies "smartphones" and uses the term "devices".

.. seealso::
   - `List of supported countries <https://help.mollie.com/hc/en-us/articles/33911501243154-In-person-payments-supported-countries>`_

Mollie configuration
====================

To configure a Mollie terminal, go to your `Mollie account <https://my.mollie.com>`_,

> and then follow these steps:

Reviewer: ```suggestion
To configure a Mollie terminal, create a `Mollie account <https://my.mollie.com>`_, then follow these steps:
```

> #. Go to :guilabel:`Point-of-sale` in the left sidebar.

Reviewer: ```suggestion
#. Go to the :guilabel:`In-person payments` section in the left sidebar to access the list of terminals.
```

Reviewer: I do not see any **Point-of-sale** section, but I see the list of terminals in the **In-person payments** section.

<img width="1916" height="485" alt="Image" src="https://github.com/user-attachments/assets/6be252c3-aa43-41c2-bbf8-6867d8a1549f" />

#. You should see your list of payment terminals. If you haven't setup a terminal with Mollie yet,

>    you can find all the available options on the `Mollie website <https://www.mollie.com/products/pos-payments>`_.

Reviewer: This should be a note below the **Warning** banner. 

```suggestion
.. note::
  To set up a Mollie terminal, find all the available options on the `Mollie website <https://www.mollie.com/products/pos-payments>`_.
```

#. Select the desired payment terminal from the list.

> #. Go to :guilabel:`Terminal information` from the tabs at the top.

Reviewer: ```suggestion
#. Click the :guilabel:`Terminal information` tab in the top menu.
```

> #. Copy the :guilabel:`Terminal ID` for later.

Reviewer: ```suggestion
#. Copy the :guilabel:`Terminal ID` and keep it handy for the :ref:`Odoo POS configuration <ADD_REF_LINK>` process.
```

> #. Go to the :guilabel:`Developers` link in the bottom left corner.

Reviewer: ```suggestion
#. Click the :guilabel:`Developers` icon in the bottom left corner.
```

> #. Copy the :guilabel:`Live API key` for later.

Reviewer: ```suggestion
#. Copy the :guilabel:`Live API key` and keep it handy for the :ref:`Odoo POS configuration <ADD_REF_LINK>` process.
```

.. warning::
   Treat your Live API key like a password, do not share it with anyone or write it down anywhere insecure,

>    as it can grant access to your Mollie account.

Reviewer: ```suggestion
   Treat the :guilabel:`Live API key` as a password and keep it secure, as it provides access to your Mollie account.
```

Odoo POS configuration
======================

To connect the Mollie terminal with Odoo Point of Sale, follow these steps:

#. Go to :menuselection:`Point of Sale --> Configuration --> Payment Methods` and :doc:`create a
   payment method <../../payment_methods>`.
#. Set the :guilabel:`Journal` field to :guilabel:`Bank`.
#. Select the desired point of sale in the :guilabel:`Point of Sale` field.
#. Set the :guilabel:`Integration` field to :guilabel:`Terminal`.
#. :guilabel:`Activate` the Mollie integration from the list of providers if you haven't already.

> #. Set the :guilabel:`Integrate with` field to :guilabel:`Mollie`.

Reviewer: ```suggestion
#. Click :guilabel:`Activate` or :guilabel:`Setup` under the Mollie integration from the list of providers if needed.
 #. Optionally, set the :guilabel:`Integrate with` field to :guilabel:`Mollie`.
```

#. Set the :guilabel:`Mollie Payment Provider` field to :guilabel:`Mollie`.
#. Paste the Terminal ID into the :guilabel:`Mollie Terminal ID` field.

> #. Save the form.

Reviewer: ```suggestion
#. Save the payment method.
```

> #. Follow the internal link on :guilabel:`Mollie Payment Provider`.

Reviewer: ```suggestion
#. Click the :icon:`fa-arrow-right` (:guilabel:`Internal link`) icon next to the :guilabel:`Mollie Payment Provider` field.
```

#. Paste the API key into the :guilabel:`API Key` field. You can leave the payment provider as

>    :guilabel:`Disabled` if you don't intend to also use it for online payments.

Reviewer: ```suggestion
#. Paste the API key into the :guilabel:`API Key` field. Leave the :guilabel:`State` field on :guilabel:`Disabled` if the payment provider is not used for online payments.
```
